cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySql/BQValidation.java

```diff
@@ -0,0 +1,225 @@
+package io.cdap.plugin.CloudMySql;
+
+import com.google.cloud.bigquery.TableResult;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.type.Decimal;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.CloudMySqlClient;
+import io.cdap.plugin.common.stepsdesign.TestSetupHooks;
+import org.junit.Assert;
+import org.junit.Test;
+import java.io.IOException;
+import java.sql.*;
+import java.sql.Date;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+/**
+ *  BQValidation
+ */
+
+public class BQValidation {
+
+    public static List<JsonObject> bigQueryResponse = new ArrayList<>();
+    public static List<Object> bigQueryRows = new ArrayList<>();
+
+    /**
+     * Extracts entire data from source and target tables.
+     * @param sourceTable table at the source side
+     * @param targetTable table at the sink side
+     * @return true if the values in source and target side are equal
+     */
+
+
+    public static boolean validateBQAndDBRecordValues(String sourceTable, String targetTable)
+            throws SQLException, ClassNotFoundException, IOException, InterruptedException, ParseException {
```

Comment:
fix the indentation/spacing.

Reply:
done
commit link- b03766131af7a687ad66f495a6eac551f74c15d8